tests/test_flows.py

```diff
@@ -646,13 +658,20 @@ def test_combined_infra_metrics(self):
                                                env.controller.mac, mid)
         debug("Check AP metrics response has AP metrics")
         ap_metrics_1 = self.check_cmdu_has_tlv_single(response, 0x94)
-        # TODO currently STA link metrics are missing
-        # debug("Check AP metrics response has STA link metrics")
-        # sta_metrics_1 = self.check_cmdu_has_tlv_single(response, 0x96)
-
+        if ap_metrics_1:
+            if ap_metrics_1.ap_metrics_bssid != vap1.bssid:
+                self.fail("AP metrics response with wrong BSSID {} instead of {}".format(
+                    ap_metrics_1.ap_metrics_bssid, vap1.bssid))
+
+        debug("Check AP metrics response has STA traffic stats")
+        sta_stats_1 = self.check_cmdu_has_tlv_single(response, 0xa2)
+        if sta_stats_1:
+            if sta_stats_1.assoc_sta_traffic_stats_mac_addr != sta1.mac:
+                self.fail("STA traffic stats with wrong MAC {} instead of {}".format(
+                    sta_stats_1.assoc_sta_traffic_stats_mac_addr, sta1.mac))
+
+        debug("Send AP Metrics query message to agent 2 with STA Metrics for {}".format(sta2.mac))
```

Comment:
Debug message is mixing AP Metrics Query message with Multi-AP Policy Config Request message

Reply:
fixed